src/context/auto-executor.test.ts

```diff
@@ -273,6 +276,7 @@ describe("auto-executor", () => {
             {
               destination: WORKFLOW_ENDPOINT,
               headers: {
+                "upstash-feature-set": "WF_NoDelete",
```

Comment:
What does this header do? Just to know

Reply:
In @upstash/qstash, if the request in context.call failed, the workflow run would fail and [failure handlers](https://upstash.com/docs/qstash/workflow/howto/failures) would be called.

Now, we don't want to make the workflow run fail, but we want to continue from where we left off. This header tells the qstash-server that this workflow is from the new sdk, and run shouldn't fail